opencti-platform/opencti-front/src/private/components/data/DataTableToolBar.jsx

```diff
@@ -1270,7 +1270,7 @@ class DataTableToolBar extends Component {
     const selectedElementsList = Object.values(this.props.selectedElements || {});
 
     const selectedTypes = R.uniq([...selectedElementsList.map((o) => o.entity_type), ...entityTypeFilterValues]
-      .filter((entity_type) => entity_type !== undefined));
+      .filter((entity_type) => entity_type !== undefined && entity_type !== 'Stix-Cyber-Observable'));
```

Comment:
It seems that we have a constant :
`const typesWithScore = ['Stix-Cyber-Observable', 'Indicator'];`
I was wondering if we could use it instead of hard coding 'Stix-Cyber-Observable' ?

Reply:
In fact, I replaced it, taking care not to impact the indicators.
`.filter((entity_type) => entity_type !== undefined && entity_type !== typesWithScore[0]));`